README.md

```diff
@@ -1,155 +1,111 @@
 # giantswarm-aws-account-prerequisites
 
-This repo contains Terraform modules to prepare AWS accounts to run Giant Swarm
-clusters.
+This repo contains Cloud Formation templates and Terraform modules to prepare AWS accounts to run Giant Swarm clusters.
 
 # Cluster API
+
 ## Before starting
-Make sure to adjust AWS account limits according to [these
-docs](https://docs.giantswarm.io/getting-started/cloud-provider-accounts/cluster-api/aws/#limits). 
 
-For Cluster API take a look at theese three modules in this repository:
-1. [admin-role](./admin-role) which creates a role and a policy for our
-   staff to be able to operate the infrastructure created by our automation in
-   case of failures.
-3. [capa-controller-role](./capa-controller-role) which creates
-   the role and policies that the controllers assume to create and manage the kubernetes clusters.
+Make sure to adjust AWS account limits according to [these docs](https://docs.giantswarm.io/getting-started/cloud-provider-accounts/cluster-api/aws/#limits).
 
-## 1. admin-role
+For Cluster API take a look at these two modules in this repository:
 
-For all AWS accounts part of the platform, does not matter if they are for
-management or workload clusters, we need to have access in order to debug and
-manage and operate the infrastructure. To do so, please run this module in the target
-account:
+1. [admin-role](./admin-role) which creates a role and a policy for our staff to be able to operate the infrastructure created by our automation in case of failures.
+2. [capa-controller-role](./capa-controller-role) which creates the role and policies that the controllers assume to create and manage the kubernetes clusters.
 
-```hcl
-module "giantswarm-cp-prereqs" {
-  source = "git@github.com:giantswarm/giantswarm-aws-account-prerequisites/admin-role"
-}
+## 1. admin-role
 
-output "giantswarm-admin-role" {
-  value = "${module.giantswarm-cp-prereqs.giantswarm-admin-role}"
-}
-```
+For all AWS accounts part of the platform  Giant Swarm staff need to have access in order to debug and
+manage and operate the infrastructure. To do so, please use one of the following methods to create the necessary role and policy in your AWS account.
 
-The created role and policy name is `GiantSwarmAdmin`.
+### AWS CloudFormation template
 
-The created role ARN needs to be supplied to Giant Swarm.
+You can execute directly the CloudFormation template just clicking [the admin role stack template](https://eu-central-1.console.aws.amazon.com/cloudformation/home?region=eu-central-1#/stacks/quickcreate?templateURL=s3://cf-templates-giantswarm/admin-role/cloud-formation-template.yaml&stackName=GiantSwarmAdminRoleBootstrap&&param_AdminRoleName=GiantSwarmAdmin) or uploading the [template file](./admin-role/cloud-formation-template.yaml) when creating a new stack in the AWS console.
```

Comment:
@paurosello I created a bucket in the root account called `cf-templates-giantswarm`. I updated the link, and now it is launching the stack creation, but I don't see parameters from the URL being parsed. Can you take a look, plz?

(I could not find your s3 bucket in any of our accounts 🤷 but it said the name is taken)